rest/bulk_api.go

```diff
@@ -364,6 +374,7 @@ func (h *handler) handleBulkGet() error {
 
 	includeAttachments := h.getBoolQuery("attachments")
 	showExp := h.getBoolQuery("show_exp")
+	showCV := h.getBoolQuery("show_cv")
```

Comment:
We should be consistent with the query parameter used to include cv in the responses.  I'm fine with `cvs=true` like you've got in the all docs case.

Reply:
I did `show_cv` somewhat arbitrarily becuase I thought it made it clearer that it is an output parameter, especially when `rev` or `revs` can be input parameters to some functions.

Also cvs reminds me of a US pharmacy or the legacy version control system. I am happy to switch to cvs though.